### DISCUSSION TITLE
Allow field/section handles in settings to auto copy

### DISCUSSION THREAD

**Discussion by knynkwl** [craftcms/cms#10475] [Ideas] [answered]
It would be nice to have the same functionality that we get on entry pages, where you can copy a filed handle, on section/field editing pages.

Field handle in entry:
<img width="171" alt="Screen Shot 2022-02-04 at 10 29 49 AM" src="https://user-images.githubusercontent.com/4225771/152556119-1fd3819d-37e3-4cd4-9094-4ae57f0df331.png">

Field handle in settings:
<img width="469" alt="Screen Shot 2022-02-04 at 10 32 04 AM" src="https://user-images.githubusercontent.com/4225771/152556489-3e8f4a12-dfe8-4c47-8e91-97b4674c3053.png">

**Comment by timkelty** [comment, marked as answer]
👀 https://github.com/craftcms/cms/pull/10489

**Reply by brandonkelly** [reply]
3.7.31 is out with that change!
